test/sql/copy/csv/test_sniff_csv.test

```diff
@@ -8,6 +8,8 @@ PRAGMA enable_verification
 # requires notwindows because tests will return \r\n to be used in the parameters
 require notwindows
 
+require httpfs
```

Comment:
Can you move te require to line 116? 
Otherwise, the whole test-file won't run when the httpfs extension is absent.

Reply:
Got it. This is why on my local machine all tests are green.